cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -114,16 +126,21 @@ func TestScan(t *testing.T) {
 			expectedMonitors: 1,
 		},
 		{
-			name: "two devices",
+			name: "pf-and-2-vfs",
 			sysfsdirs: []string{
 				"card0/device/drm/card0",
 				"card1/device/drm/card1",
+				"card2/device/drm/card2",
 			},
 			sysfsfiles: map[string][]byte{
-				"card0/device/vendor": []byte("0x8086"),
-				"card1/device/vendor": []byte("0x8086"),
+				"card0/device/vendor":       []byte("0x8086"),
+				"card0/device/sriov_numvfs": []byte("2"),
+				"card1/device/vendor":       []byte("0x8086"),
+				"card1/device/physfn":       []byte("1"),
+				"card2/device/vendor":       []byte("0x8086"),
+				"card2/device/physfn":       []byte("1"),
```

Comment:
"card2/device/physfn" doesn't seem to play any role in the tests.

BTW, is it possible that the content of this file is something other than "1"? I'm wondering what that could mean. A number of tiles on the chip?

Reply:
AFAIK its precense is one of the more common ways of identifying VFs (and I have some private GPU plugin changes which check for presence of this file), so I think it's relevant to have them, although current PR doesn't add any checks for their presence.  What do you think?